gulpfile.js

```diff
@@ -600,27 +598,21 @@ function buildExtensionJs(path, name, version, options) {
     throw new Error('Unsupported loadPriority: ' + options.loadPriority);
   }
   const priority = options.loadPriority ? 'p:"high",' : '';
-  return compileJs(path + '/', filename, './dist/v0', {
-    watch: options.watch,
```

Comment:
`compileJs` uses `options.watch` to determine whether to call `rebundle` or not. See https://github.com/ampproject/amphtml/blob/master/gulpfile.js#L1087=L1107

Will `gulp`, `gulp watch`, `gulp build`, and `gulp dist` all continue to work with this change? Also, will they work with `--noextensions` and `--extensions`?

Reply:
`options.watch`, `options.minify` and others are retained due to `Object.assign`.